L1TriggerConfig/Utilities/src/L1MenuWriter.cc

```diff
@@ -26,6 +27,28 @@ class L1MenuWriter : public edm::EDAnalyzer {
 void L1MenuWriter::analyze(const edm::Event& iEvent, const edm::EventSetup& evSetup){
 
     edm::ESHandle<L1TUtmTriggerMenu> handle1;
+    evSetup.get<L1TUtmTriggerMenuRcd>().get( handle1 ) ;
+    boost::shared_ptr<L1TUtmTriggerMenu> ptr1(new L1TUtmTriggerMenu(*(handle1.product ())));
+
+    edm::Service<cond::service::PoolDBOutputService> poolDb;
+    if( poolDb.isAvailable() ){
+        cond::Time_t firstSinceTime = poolDb->beginOfTime();
+        poolDb->writeOne(ptr1.get(),firstSinceTime,"L1TUtmTriggerMenuRcd");
+    }
+
+}
+
+class L1MenuWriter_ : public edm::EDAnalyzer {
```

Comment:
ending a class with an _ is against cms conventions - is this meant to be a new version of L1MenuWriter? If so, can we just update it instead?